airbyte-integrations/connectors/source-airtable/source_airtable/spec.json

```diff
@@ -9,18 +9,21 @@
     "properties": {
       "api_key": {
         "type": "string",
-        "description": "The API key for the Airtable account. https://support.airtable.com/hc/en-us/articles/219046777-How-do-I-get-my-API-key-",
+        "description": "The API Key for the Airtable account. See the <a href=\"https://support.airtable.com/hc/en-us/articles/219046777-How-do-I-get-my-API-key-\">Support Guide</a> for more information on how to obtain this key.",
+        "title": "API Key",
         "airbyte_secret": true,
         "examples": ["key1234567890"]
       },
       "base_id": {
         "type": "string",
-        "description": "The base ID to integrate data from",
+        "description": "The Base ID to integrate the data from",
+        "title": "Base ID",
         "examples": ["app1234567890"]
       },
       "tables": {
         "type": "array",
```

Comment:
does the `items` property need to be set with type string?

Reply:
Updated:
"items": {
          "type": "string"
        },